ui/packages/consul-ui/tests/acceptance/dc/services/show/topology/stats.feature

```diff
@@ -33,4 +33,50 @@ Feature: dc / services / show / topology / stats
     ---
     And I see topology on the tabs
     And I see the "[data-test-topology-metrics-stats]" element
+  Scenario: Given metrics is enabled, metrics stats are disabled for an ingress gateway Topology
+    Given 1 datacenter model with the value "dc1"
+    Given a "prometheus" metrics provider
+    And 1 node models
+    And 1 service model from yaml
+    ---
+    - Service:
+        Name: ingress-gateway
+        Kind: ingress-gateway
+        ID: ingress-gateway-with-id
+    ---
+    When I visit the service page for yaml
+    ---
+      dc: dc1
+      service: ingress-gateway
+    ---
+    And I see topology on the tabs
+    And I don't see the "[data-test-topology-metrics-stats]" element
+    And I see the "[data-test-sparkline]" element
+  Scenario: Given metrics is enabled, metric stats are disabled for ingress gateway as downstream services
+    Given 1 datacenter model with the value "dc1"
+    Given a "prometheus" metrics provider
+    And 1 node models
+    And 1 service model from yaml
+    ---
+    - Service:
+        Name: service-0
+        ID: service-0-with-id
+    ---
+    And 1 topology model from yaml
+    ---
+      Upstreams: []
+      Downstreams:
+        - Name: db
+          Namespace: @namespace
+          Datacenter: dc1
+          Intention: {}
+    ---
```

Comment:
`Given metrics is enabled, metric stats are disabled for ingress gateway as downstream services`

I'm not sure if I can see here that we are setting up the test to test an ingress-gateway? I ran the test and I see this:

<img width="641" alt="Screenshot 2021-08-25 at 11 40 17" src="https://user-images.githubusercontent.com/554604/130776559-63fe1e24-d7df-4f82-92e7-af38665ec362.png">

Is `service-0` or `db` the ingress-gateway here?

I also noticed the "No metrics available" message here which is different to the screengrabs in your description? I'm not sure whether its supposed to show this message or not? I think that if db is the ingress, then judging from the screengrabs in the description I don't think this message is supposed to show? Not sure.

Reply:
`db` is the ingress-gateway here. I've added `Kind: ingress-gateway` to it. A typo didn't catch this. I've update the data-test to say `data-test-topology-metrics-downstream-stats`.